README.md

```diff
@@ -12,24 +12,40 @@ A [Magento](http://magento.com/ "eCommerce Software & eCommerce Platform Solutio
 5. Done, move to configuration section.
 
 ## Configuration
-Extension configuration can be found from Magento administration interface, under `System` &rarr; `Configuration` &rarr; `Newsletter` &rarr; `Sendsmaily synchronization` section.
+Extension configuration can be found from Magento administration interface, under `System` &rarr; `Configuration` &rarr; `Newsletter` &rarr; `Smaily Email Marketing And Automation` section.
 
-* **Enabled** &minus; Enables the extension; manual and regular export. Defaults to "No".
-* **Sendsmaily account** &minus; Subdomain part of the Sendsmaily account. For example `xxx` from `https://xxx.sendsmaily.net/`.
-* **API key** &minus; Sendsmaily account's API key.
-* **Additional data** &minus; Additional data related to customer. By default only email address and subscribed state is exported.
-* **How often do you want the cron to run?** &minus; Export subscriber data on a regular basis. To disable regular exports leave the field empty. Defaults to once every day at 1:30 in the morning.
+* **General settings** &minus; Enables the extension and holds credentials related information.
+* **Customer Synchronization** &minus; Manages subscriber synchronization settings. Select fields to sync and sync frequency.
+
+## Usage
+
+1. Go to `System` &rarr; `Configuration` &rarr; `Newsletter` &rarr; `Smaily Email Marketing And Automation`.
+2. Select `Enabled` &rarr; `Yes` status in General Settings
+3. Insert your Smaily API authentication information and press `Save Config` to get started
+4. To enable customer synchronization select `Enabled` &rarr; `Yes` status in Customer Synchronization Settings
+5. Select fields you would like to sync and frequency
 
 ## Manual export
 At any time you can trigger a manual export from `Newsletter` &rarr; `Newsletter subscribers` by selecting `Sendsmaily` from `Export to:` dropdown and hitting that `Export` button.
 
-To export specific subscriber(s), filter desired subscribers using the fields under the table header and click `Export`. **NOTE! All filtered subscribers are exported, selecting/marking row(s) does not export that/these row(s).**
+To export specific subscriber(s), filter desired subscribers using the fields under the table header and click `Export`. 
+**NOTE! All filtered subscribers are exported, selecting/marking row(s) does not export that/these row(s).**
```

Comment:
The wording of this note is cumbersome. I suggest changing it to something like:

> NOTE! All filtered subscribers are always exported. Row selection has no effect.

Reply:
Changed this comment in 608fbd51c1b00c0b7b2dfcd11dda1b068a5c5e16